syntax = "proto3";

message EchoRequest {
  string msg = 1;
}

message EchoReply {
  string msg = 1;
}

service EchoService {
  rpc Echo(EchoRequest) returns (EchoReply) {}
}
